2/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/rhscl/s2i-core-rhel7
+FROM registry.access.redhat.com/rhel7
```

Comment:
we probably don't even need the `s2i-core-rhel7` base image and plain `rhel7` works